example/app/config.yml

```diff
@@ -731,15 +732,34 @@ mhs4:
   ecs:
     ssh: true
     keyPairName: mhs
+  launchpad:
+    api: 'https://api.launchpad.nasa.gov/icam/api/sm/v1'
+    passphrase: '{{LAUNCHPAD_PASSPHRASE}}'
   apiConfigs:
     backend:
       private: true
       port: 8004
     distribution:
       private: true
-      port: 7004
+      port: 7014
       logApiGatewayToCloudWatch: true
   logToSharedDestination: '{{SHARED_LOG_DESTINATION_ARN}}'
+  cmr:
+    oauthProvider: launchpad
+    username: CmrUserNotUsed
+    password: CmrPasswordNotUsed
+  oauth:
+    provider: launchpad
+    userGroup: GSFC-Cumulus-Dev
+  # oauth:
+  #   provider: earthdatalogin
+  #   userGroup: n/a
+  samlConfig:
```

Comment:
Are there default values for any of this that should be added to the deployment package?

Reply:
Not sure.  Maybe dummy values?

Reply:
Don't think there are any useful defaults.  Maybe use dummy values?

Reply:
Also, I don't know how we would be able to get the api endpoint during deployment, but maybe I should think about that.